packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -259,7 +260,10 @@ const Updates = decorate([
           .join('\n'),
       proxyFormId: generateId,
       xoaBuild: async () => {
-        const { build = 'unknown' } = await getApplianceInfo()
+        const { build = 'unknown' } = defined(
+          await getApplianceInfo()::ignoreErrors(),
+          {}
+        )
```

Comment:
IMHO this is simpler:

```js
const { build = 'unknown' } = await getApplianceInfo().catch(error => {
  console.warn('getApplianceInfo', error)
  return {}
})
```

You could even implement this in `getApplianceInfo` if you prefer.

Reply:
I'd prefer to stay in the PR's scope for now.